### ISSUE TITLE
Application pages

### ISSUE THREAD

**Issue by LuisF3** [flossschool/pollex#20]
This issue is for discussing and developing the remaining pages of the application. My group is currently working on it, but any help is welcome :)

As soon as possible we will release the designs here

**Comment by Vakihito** [comment]
So we changed the homepage to fit monaco's especification, and that's what we have so far:
![image](https://user-images.githubusercontent.com/40372914/69918721-7458b380-1454-11ea-8118-f0c756c94be4.png)
the color represents the type of pool you have

**Comment by henritarzia** [comment]
We started the poll edit template. It isn't complete yet, but we will add inputs related to the voting options and title.
![Poll template](https://user-images.githubusercontent.com/44980986/69919018-622c4480-1457-11ea-9cc0-65f8150b1bbd.png)

**Comment by henritarzia** [comment]
I finished to make the poll edit and visualization templates. I thought it would be interesting to add the option to include administrators that can help inviting people and adding voting options in the poll besides the creator, so I added a field in the form for this. Any suggestions to modify the templates are welcome. 

![image](https://user-images.githubusercontent.com/44980986/69979760-a8dc7600-150d-11ea-9527-ece2b9086b12.png)

![image](https://user-images.githubusercontent.com/44980986/69979775-b1cd4780-150d-11ea-8a3b-e73f2ee7f3f2.png)
